src/app/zap-templates/zcl/data-model/chip/administrator-commissioning-cluster.xml

```diff
@@ -38,6 +38,11 @@ limitations under the License.
     <define>ADMINISTRATOR_COMMISSIONING_CLUSTER</define>
     <description>Commands to trigger a Node to allow a new Administrator to commission it.</description>
 
+    <features>
```

Comment:
From what I understand the new xml comes from https://github.com/project-chip/connectedhomeip/tree/master/data_model . I have used the new xml to update the existing xml since @tecimovic is still working on updating ZAP to the new xml from what I know. The only reason I updated the existing xml is so that we can start having the warnings for existing apps without having to wait for the new xml. Made sure the formatting comes from the new xml so functionality remains the same as much as possible when we make permanent switch.

Reply:
I would have expected a zap update to be included together with this metadata update, so that we can verify that the new information takes effect. I think when we do that, we should remove the old feature bitmaps and use these new items instead. At that time we should verify that code generation gives no differences.

As it stands, this duplicates information and having two "sources of truth" is unlikely to stay in sync (in this case I think these new "feature" entries will just be ignored by all tooling we have)

Reply:
I can pretty easily add a parameter to alchemy to auto-generate these and remove the old ones, when we're ready on the ZAP side.

Reply:
@brdandu My point was that we should not have both this bit and the Feature bitmaps in the "old" XML (which is the only XML used for anything at the moment).

Reply:
OK, released v0.4.8 of alchemy, which has a new "featureXML" parameter that will generate these directly from the spec, and automatically remove the old feature bitmap. It also includes the new feature XML format on every cluster, not just the first one in each file. This parameter is off by default.

Reply:
I can check on the removal of the old feature bit information in the xml. Not sure if I will break something or not by removing that information. Also do not know where all in the generation that information is currently used.
However I do want to say that we should not withhold features which wait on the new xml. If we have a feature why not enable it via the old xml?
I can start looking into removing the old feature bit data from the xml and see what happens

Reply:
Oh, it'll 100% break; if the zap tool hasn't been updated to look for this new feature schema, and the old feature schema has been removed, it'll fail. That's why I put it in as a flag to use once whatever has to happen in the zap tool has happened.

One weird aspect of the new approach here is that it's just pulling over the conformance schema from the data model side. With the existing approach the zap tool doesn't get into the weeds on feature-based conformance; everything boils down to "is this mandatory or not mandatory?" via an "optional" attribute.

Reply:
Ok I see that <bitmap name="Feature"...> has the same data as <features> but as of now <bitmap> is considered as one of the data types required by the cluster and it is not equivalent to <features>. features is not considered as a data type but a separate entity in the ZAP schema since it is not like any other bitmap relationship we have. Could we keep them separate for now and deal with combining the two in a separate PR?

Reply:
I think Boris' point was: "why add this to the XML now when nothing can use it?" We should update the ZAP tool to understand the new schema (as well as the old as a fallback), and then we can add the new schema to the XML while removing the old and bumping the min ZAP version. Having both just means something else that could fall out of sync.

Reply:
@brdandu I think the basic point is: we do not want the manual maintenance burden of maintaining these two sets of data.  And since only one of them leads to visible anything on the Matter side, that means that the other will break as soon as someone adds a new feature bit to a spec, because no one will realize it should be added to that part of the XML.

If there is a concrete commitment with a timeline to not having the two things around at once, and some urgency to the UI behavior you describe, we can have the conversation about doing things this way and then making it right in a followup.  But we would need to understand the timelines involved to even start that conversation.

Reply:
Alright I added the ZAP changes required for this in https://github.com/project-chip/zap/pull/1300 and have removed the bitmap name='feature' from the xml. however the zap PR needs to be updated before this can be merged in.